app/models/screenshot.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# The Screenshot class represents the screenshot model
+class Screenshot < ApplicationRecord
+  belongs_to :locales
```

Comment:
Follow the same comment https://github.com/lingostack/lingoqa/pull/2/files?file-filters%5B%5D=.png&file-filters%5B%5D=.rb&hide-deleted-files=true#r350658393